osmosis/chain.json

```diff
@@ -506,6 +506,9 @@
       {
         "denom": "ibc/FE2CD1E6828EC0FAB8AF39BAC45BC25B965BA67CCBC50C13A14BD610B0D1E2C4"
       },
+      {
+        "denom": "ibc/CD6412358F33B372A355CF22786D8C19477C15092B56BD56188679EED8556964"
```

Comment:
Please do not add this. the new DGN isn't yet accepted as a fee token. It has to be added by Govenrnace